homeassistant/components/androidtv/media_player.py

```diff
@@ -475,6 +477,32 @@ def unique_id(self):
         """Return the device unique id."""
         return self._unique_id
 
+    async def async_get_media_image(self):
+        """Fetch current playing image."""
+        media_data = self.get_raw_media_data()
+        if media_data:
+            return binascii.a2b_base64(media_data), "image/png"
```

Comment:
Why are we encoding it to base64 and then decoding it again ? 

Also, let's do this inside `get_raw_media_data` so that it's not done in the event loop.

Reply:
Base64 is the only way to get it via ADB (that I know of); other than doing something more dodgy (e.g saving to a file on the Android TV and then getting that somehow).

Reply:
@balloob ... I know this has already been merged in ... but to elaborate on what I meant before...
Basically you can (in theory) get the screenshot as raw data via adb without going to Base64 ... but all my research seems to point to this being somewhat hit-and-miss ...
Some versions of android appear to put line breaks in the output, others do other strange stuff with replacing some bytes with 2 bytes ..
I imagine that there are probably some limitations on what data can be output via the adb or console.